readme.md

```diff
@@ -154,6 +154,28 @@ alfy.cache.get('unicorn');
 //=> '🦄'
 ```
 
+##### maxAge
+
+The `set` method of this instance accepts an optional third argument where you can provide a `maxAge` option. `maxAge` is
+the number of milliseconds the value is valid in the cache.
+
+Example:
+
+```js
+const delay = require('delay');
```

Comment:
I prefer a require at the top instead of `await require('delay')(5000)`. But if you want it inline, I'll fix it :).